API/ev3_lcd.c

```diff
@@ -705,13 +705,19 @@ bool LcdUpdate()
 	return true;
 }
 
+static short X0 = 0;
+static short Y0 = 0;
+static short CURSOR_X = 0;
+static short CURSOR_Y = 0;

```

Comment:
These are the variables used in LcdPrintf and a handful of other functions that keep track of the cursor position.